### DISCUSSION TITLE
How do I play this whole animation as one scene?

### DISCUSSION THREAD

**Discussion by MathLoverGamingMatrixes** [3b1b/manim#1605] [General]
The code -

`from manim import *

class FirstScene(Scene):
    def construct(self):

        text = Text("An alternative to integration!")
        boundary = AnimatedBoundary(text, colors=[RED, GREEN, BLUE],
                                    cycle_rate=3)
        text2 = Text("By Aaron")

        text3 = Text("Let's consider a graph of \\X^2")

        self.play(FadeIn(text, boundary))
        self.wait(2)
        self.play(ReplacementTransform(text, text2))
        self.wait(2)
        self.play(FadeIn(text3))

#graph scene
class GraphScene(Scene):
    def construct(self):

        axes = Axes(
            x_range=[-3, 3],
            y_range=[-5, 5],
            axis_config={"color": BLUE},
        )

        # Create Graph
        graph = axes.get_graph(lambda x: x ** 2, color=WHITE)
        graphlabel = axes.get_graph_label(graph, label='x^{2}')

        graph2 = axes.get_graph(lambda x: x ** 3, color=WHITE)
        graph_label2 = axes.get_graph_label(graph2, label='x^{3}')

        # Display graph after text
        self.wait(2)
        self.play(Create(axes), Create(graph), Write(graphlabel))
        self.wait(3)
       
`

**Comment by TonyCrane** [comment]
I don't understand what you are going to do.

And I noticed that you are using the community version of manim, so please move this to their discord channel.

**Reply by MathLoverGamingMatrixes** [reply]
My question is - In the code above, there are two separate scenes (the FirstScene and the GraphScene), how do I merge the two scenes together to make it one whole animation because when I try to run the code I can only run the two scenes **separately**

**Reply by TonyCrane** [reply]
If you mean that you want the two scenes to play at the same time, then best way is to use video editing software to merge them.

btw, why you open this discussion in the general category not the Q&A category. And I said the version you use is not 3b1b/manim, so you'd better not ask the question here.

**Reply by MathLoverGamingMatrixes** [reply]
So so sorry. I am new to manim and GitHub and I saw questions here so I thought I could ask. If you don’t mine me just asking one last thing, how do I get 3b1b manim? Thanks for the advice btw.

**Reply by TonyCrane** [reply]
Just follow the doc: https://3b1b.github.io/manim
The community version is also good, but you need to read their doc: https://docs.manim.community.
